core/src/main/java/org/elasticsearch/action/search/SearchPhaseController.java

```diff
@@ -459,12 +459,17 @@ private SearchHits getHits(ReducedQueryPhase reducedQueryPhase, boolean ignoreFr
             reducedQueryPhase.maxScore);
     }
 
+    public final ReducedQueryPhase reducedQueryPhase(List<? extends AtomicArray.Entry<? extends QuerySearchResultProvider>> queryResults) {
+        return reducedQueryPhase(queryResults, null);
+    }
+
     /**
      * Reduces the given query results and consumes all aggregations and profile results.
      * @see QuerySearchResult#consumeAggs()
      * @see QuerySearchResult#consumeProfileResult()
      */
-    public final ReducedQueryPhase reducedQueryPhase(List<? extends AtomicArray.Entry<? extends QuerySearchResultProvider>> queryResults) {
+    public final ReducedQueryPhase reducedQueryPhase(List<? extends AtomicArray.Entry<? extends QuerySearchResultProvider>> queryResults,
+                                                     List<InternalAggregations> reducedAggs) {
```

Comment:
Can you add a comment to explain that reducedAggs is the result from intermediate reduce operations?